src/main/java/org/jenkinsci/plugins/awsdevicefarm/AWSDeviceFarmGraph.java

```diff
@@ -121,7 +121,7 @@ public static Graph createResultTrendGraph(AbstractBuild<?, ?> owner, Boolean is
         List<NumberOnlyBuildLabel> cols = new ArrayList<NumberOnlyBuildLabel>();
 
         for (AWSDeviceFarmTestResult result : results) {
-            AbstractBuild<?, ?> build = result.getOwner();
+            hudson.model.Run<?, ?> build = result.getOwner();
```

Comment:
I don't see a reason why you can't import this at the top and just reference it as `Run`. Don't need a namespace slash.